source/_docs/terminus/examples.md

```diff
@@ -129,6 +130,53 @@ The following command will permanently delete all uncommitted SFTP changes. If y
 <figure><pre id="mass-update-apply"><code class="command bash" data-lang="bash">terminus site:list --format=list | terminus site:mass-update:apply --accept-upstream --updatedb</code></pre></figure>
 </div>
 
+## Switch Upstreams
+Every site has an upstream assigned in order to deliver [one-click updates](/docs/upstream-updates/) in the Pantheon Site Dashboard. Terminus can be used to manage this site level configuration.
```

Comment:
I know this is still in progress, but this intro paragraph would be a good time to give a few example use cases we know users have from the research, to make this more concrete and help users understand why they might want to do this.